core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -114,54 +115,61 @@ static void initialize(PageStoreOptions options) throws IOException {
   /**
    * Writes a new page from a source channel to the store.
    *
-   * @param pageId page identifier
-   * @param page page data
+   * @param pageId   page identifier
+   * @param page     page data
+   * @param pageInfo page info
    * @throws ResourceExhaustedException when there is not enough space found on disk
-   * @throws IOException when the store fails to write this page
+   * @throws IOException                when the store fails to write this page
    */
-  void put(PageId pageId, byte[] page) throws ResourceExhaustedException, IOException;
+  void put(PageId pageId, byte[] page, PageInfo pageInfo)
+      throws ResourceExhaustedException, IOException;
 
   /**
    * Gets a page from the store to the destination buffer.
    *
-   * @param pageId page identifier
-   * @param buffer destination buffer
+   * @param pageId                 page identifier
+   * @param lastModificationTimeMs last modified time of this page
+   * @param buffer                 destination buffer
    * @return the number of bytes read
-   * @throws IOException when the store fails to read this page
+   * @throws IOException           when the store fails to read this page
    * @throws PageNotFoundException when the page isn't found in the store
    */
-  default int get(PageId pageId, byte[] buffer) throws IOException, PageNotFoundException {
-    return get(pageId, 0, buffer.length, buffer, 0);
+  default int get(PageId pageId, long lastModificationTimeMs, byte[] buffer)
```

Comment:
it is weird to also pass lastModificationTime in get here.
Can we check and compare lastModificationTime in `LocalCacheManager`, without further passing down to page store? we can talk this offline